### ISSUE TITLE
Didn't work with dynamic wallpaper from saint-13

### ISSUE THREAD

**Issue by blankov** [avanishsubbiah/material-you-theme#27]
**Hello.**

**Dynamic wallpaper that i use:** https://github.com/saint-13/Linux_Dynamic_Wallpapers

**Error:**
![Screenshot from 2022-08-07 18-14-02](https://user-images.githubusercontent.com/47553128/183298087-edd2b0af-04c1-437c-99eb-8ad54c87f20e.png)

**How the wallpaper setting looks in Gnome Tweaks:**
![Screenshot from 2022-08-07 18-14-19](https://user-images.githubusercontent.com/47553128/183298230-9f742e3c-2719-4d8f-8bf9-06fa9dea5f48.png)

**Comment by avanishsubbiah** [comment]
Yeah I'm using GdkPixBuf() to read a array of pixels from the wallpaper. I don't know if this works with xml files.
